pkg/snmp/snmpparse/config_snmp.go

```diff
@@ -0,0 +1,102 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+package snmpparse
+
+import (
+	"encoding/json"
+	"fmt"
+
+	yaml "gopkg.in/yaml.v2"
+
+	"github.com/DataDog/datadog-agent/cmd/agent/api/response"
+	"github.com/DataDog/datadog-agent/pkg/api/util"
+	"github.com/DataDog/datadog-agent/pkg/autodiscovery/integration"
+	"github.com/DataDog/datadog-agent/pkg/config"
+)
+
+var configCheckURLSnmp string
+
+// SNMPConfig is a generic container for configuration data specific to the SNMP
+// integration.
+
+type SNMPConfig struct {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I think we can reused existing snmp config structure [here](https://github.com/DataDog/datadog-agent/blob/435aec2fab2e093b1b0cea8b22930828d568c0be/pkg/collector/corechecks/snmp/internal/checkconfig/config.go#L74-L93) instead of redefining it.

Since [config.go](https://github.com/DataDog/datadog-agent/blob/435aec2fab2e093b1b0cea8b22930828d568c0be/pkg/collector/corechecks/snmp/internal/checkconfig/config.go#L74-L93) is located in a internal folder `pkg/collector/corechecks/snmp/internal/checkconfig` (can't be used externally), a possibility is to move `InstanceConfig` to https://github.com/DataDog/datadog-agent/tree/435aec2fab2e093b1b0cea8b22930828d568c0be/pkg/collector/corechecks/snmp/common/config.go for example.